universe/Condition.cpp

```diff
@@ -3279,8 +3280,14 @@ std::string Condition::PlanetEnvironment::Description(bool negated/* = false*/)
         }
     }
     std::string species_str;
-    if (m_species_name)
-        species_str = m_species_name->Description();
+    if (m_species_name) {
+        if (ValueRef::ConstantExpr(m_species_name)) {
+            species_str = boost::lexical_cast<std::string>(m_species_name->Eval());
```

Comment:
m_species_name->Eval() should return std::string so I don't think there's any need to cast to std::string...